apps/dh/api-dh/source/DataHub.WebApi/Controllers/MarketParticipantUserRoleController.cs

```diff
@@ -25,39 +27,90 @@ namespace Energinet.DataHub.WebApi.Controllers
     [Route("v1/[controller]")]
     public class MarketParticipantUserRoleController : MarketParticipantControllerBase
     {
-        private readonly IMarketParticipantUserRoleClient _client;
+        private readonly IMarketParticipantUserRoleClient _userRoleClient;
+        private readonly IMarketParticipantClient _client;
 
-        public MarketParticipantUserRoleController(IMarketParticipantUserRoleClient client)
+        public MarketParticipantUserRoleController(IMarketParticipantUserRoleClient userRoleClient, IMarketParticipantClient client)
         {
+            _userRoleClient = userRoleClient;
             _client = client;
         }
 
         [HttpGet]
         [Route("Get")]
         public Task<ActionResult<IEnumerable<UserRoleDto>>> GetAsync(Guid actorId, Guid userId)
         {
-            return HandleExceptionAsync(() => _client.GetAsync(actorId, userId));
+            return HandleExceptionAsync(() => _userRoleClient.GetAsync(actorId, userId));
         }
 
         [HttpGet]
         [Route("GetAll")]
         public Task<ActionResult<IEnumerable<UserRoleInfoDto>>> GetAllAsync()
         {
-            return HandleExceptionAsync(() => _client.GetAllAsync());
+            return HandleExceptionAsync(() => _userRoleClient.GetAllAsync());
         }
 
         [HttpGet]
         [Route("GetAssignable")]
         public Task<ActionResult<IEnumerable<UserRoleDto>>> GetAssignableAsync(Guid actorId)
         {
-            return HandleExceptionAsync(() => _client.GetAssignableAsync(actorId));
+            return HandleExceptionAsync(() => _userRoleClient.GetAssignableAsync(actorId));
+        }
+
+        [HttpGet]
+        [Route("GetUserRoleView")]
+        public async Task<ActionResult<UserRoleView>> GetUserRoleViewAsync(Guid userId)
+        {
+            var allOrganizations = await _client.GetOrganizationsAsync();
```

Comment:
This could be formatted better, or maybe split into several methods to make it more readable.